MapboxNavigation/MGLMapView.swift

```diff
@@ -44,13 +44,13 @@ extension MGLMapView {
         }
         
         let shaftLength = max(min(50 * metersPerPoint(atLatitude: maneuverCoordinate!.latitude), 50), 10)
-        let shaftCoordinates = polyline(along: polylineCoordinates!, within: -shaftLength / 2, of: maneuverCoordinate!)
-            + polyline(along: polylineCoordinates!, within: shaftLength, of: maneuverCoordinate!)
+        let shaftCoordinates = Array(polyline(along: polylineCoordinates!, within: -shaftLength / 2, of: maneuverCoordinate!).reversed()
+            + polyline(along: polylineCoordinates!, within: shaftLength, of: maneuverCoordinate!).suffix(from: 1))
```

Comment:
mapbox/mapbox-gl-native#8808 (which is in the v3.6.0 betas) makes the map SDK more resilient to redundant vertices, but it’s good to address the root cause too. 👍